packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/RegistryPreset/Custom.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { Select, Input, Text } from '@codesandbox/components';
+import { AuthType, RegistryType } from 'app/graphql/types';
+import { RegistryPresetProps } from '.';
+import { CustomFormField } from '../RegistryForm';
+
+export const CustomRegistryPreset = ({
+  registryUrl,
+  setRegistryUrl,
+  disabled,
+  authenticationType,
+  setAuthenticationType,
+  authKey,
+  setAuthKey,
+  setRegistryType,
+}: RegistryPresetProps) => {
+  React.useEffect(() => {
+    setRegistryType(RegistryType.Custom);
+  }, [setRegistryType]);
+  return (
+    <>
+      <div>
+        <CustomFormField label="Registry URL">
+          <Input
+            value={registryUrl}
+            onChange={e => setRegistryUrl(e.target.value)}
+            required
+            pattern="https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)"
+            disabled={disabled}
+          />
+        </CustomFormField>
+        <Text size={3} variant="muted">
+          Is your registry behind a VPN? Please read these{' '}
+          <a
```

Comment:
Nice! Will change it.

Reply:
Hmm, in this case it will show as normal text:

![image](https://user-images.githubusercontent.com/587016/114558986-8d3dbf80-9c6b-11eb-8358-eab47f928768.png)

However, it does have a nice hover:

![image](https://user-images.githubusercontent.com/587016/114559017-962e9100-9c6b-11eb-9097-4acc0606b852.png)

But people won't be able to see that it really is a URL, I changed the `variant` to `active` and it shows this:

![image](https://user-images.githubusercontent.com/587016/114559231-cc6c1080-9c6b-11eb-9235-b35996d1c77f.png)

It feels like it screams a bit, but it fits our colors better. What do you think of this version @DannyRuchtie ?